static/js/components/capability/capabilities.js

```diff
@@ -0,0 +1,56 @@
+/**
```

Comment:
The `components` directory seems like a bit of a strange place for this.

Reply:
Maybe this code should just be merged with the other new file?

Reply:
I placed it there because all the custom element definitions are located there.
But I think it makes sense to merge it with the other file as it also translates the capability into something else.